demo/src/main/java/com/novoda/landingstrip/SimpleTextTabActivity.java

```diff
@@ -39,8 +39,8 @@ public void onTabClicked(int position, Data data) {
             }
         };
         ExampleTabAdapter tabAdapter = new ExampleTabAdapter(getLayoutInflater(), setCurrentItemOnClick, data);
-
-        viewPager.addOnPageChangeListener(landingStrip);
+        ViewPager.OnPageChangeListener landingStripPageChanger = LandingStripPageChangeListener.create(landingStrip);
+        viewPager.addOnPageChangeListener(landingStripPageChanger);
```

Comment:
It would avoid exposing some package methods, these are our options 
```
viewPager.addOnPageChangeListener(landingStrip);
```
```
viewPager.addOnPageChangeListener(landingStrip.pageChangeListener());
```
```
viewPager.addOnPageChangeListener(LandingStripPageChangeListener.create(landingStrip);
```